.changeset/happy-toys-boil.md

```diff
@@ -0,0 +1,15 @@
+---
+'@sveltejs/vite-plugin-svelte': major
+---
+
+feat: convert to es module and provide cjs fallback
+
+If you are using vite-plugin-svelte with require, you should switch to esm and use import.
+An example can be found in the usage section of the [readme](README.md)
+
+continuing with cjs/require is discouraged but if you must use it, update your require statement to add `.default`
+
+```diff
+- const svelte = require('@sveltejs/vite-plugin-svelte');
++ const svelte = require('@sveltejs/vite-plugin-svelte').default;
```

Comment:
is

```
const { default: svelte } = require('@sveltejs/vite-plugin-svelte');
```

easier on the eye?

Reply:
got a slight preference for the former hence i went with it. But if you feel the {default: svelte} syntax is more modern or has other benefits i'm open for changing it. 

cjs is a fallback anyways and i'd love to make a total switch. more on that below....

Reply:
named export solved this